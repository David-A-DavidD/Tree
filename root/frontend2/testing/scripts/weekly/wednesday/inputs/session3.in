login
admin
list
listusers
logout
exit