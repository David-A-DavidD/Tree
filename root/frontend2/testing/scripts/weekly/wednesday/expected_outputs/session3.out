Enter transaction code (or 'exit' to quit): Enter username: Login successful.
Available Games:
Game Name                    Seller           Price
mygame_____________________   admin           100.00
----------------------------------------------------
Mario______________________   admin           100.00
----------------------------------------------------
halo3______________________   admin           100.00
----------------------------------------------------
gameForSale________________   sellUser        100.00
----------------------------------------------------
User Information:
Username           User Type     Credit
admin___________   AA           1600.00
-----------------------------
test____________    FS           360.00
-----------------------------
Justin__________    AM          1620.00
-----------------------------
sellUser________    SS          1620.00
-----------------------------
buyUser_________    BS          1620.00
-----------------------------
Logout successful.